sdk/typescript/runtime/main.go

```diff
@@ -648,7 +650,7 @@ func (t *TypescriptSdk) installDependencies(ctr *dagger.Container) (*dagger.Cont
 			WithExec([]string{"npm", "ci"}), nil
 	case BunManager:
 		return ctr.
-			WithExec([]string{"bun", "install", "--no-verify", "--no-progress"}), nil
+			WithExec([]string{"bun", "install", "--no-verify", "--no-progress", "--frozen-lockfile"}), nil
```

Comment:
`--frozen-lockfile` may break the runtime since some additional install may happen like the `sdk` dir, if all tests are failing, you can try to remove this option and see if it fix the problem.

Reply:
ah okay, I just saw frozen lockfile being used on yarn and assumed Bun's logic should match.